satellite/console/consoleweb/server.go

```diff
@@ -214,6 +214,8 @@ func (server *Server) appHandler(w http.ResponseWriter, r *http.Request) {
 	header.Set(contentType, "text/html; charset=UTF-8")
 	header.Set("Content-Security-Policy", strings.Join(cspValues, "; "))
 	header.Set("X-Content-Type-Options", "nosniff")
+	header.Set("X-Frame-Options", "DENY")        // Prevent being included in a <frame> or <iframe>
```

Comment:
hm...but we have a universal login page that renders login and register pages in Iframe

Reply:
oh neat. I didn't know that was a thing. I updated it to support the tardigrade.io domain. We'll want to move this to a config setting at some point.